tools/cdk-build-tools/config/nycrc

```diff
@@ -9,6 +9,7 @@
         "lcov",
         "text-summary"
     ],
+    "extension": [".js", ".ts"],
```

Comment:
I think we just need `.js`

Reply:
Also not sure about specifying both extensions

Reply:
I think just .js

Reply:
With just `.js` we are back to `Unknown%`. It works with only `.ts`...

Reply:
But globally the coverage is lower than before